_data/banking.yml

```
@@ -163,6 +163,14 @@ websites:
       phone: Yes
       doc: https://www.mybankofinternet.com/tob/live/usp-core/static/help.html#login_security
 
+    - name: Boursorama
+      url: https://www.boursorama.com/
+      twitter: Boursorama
+      facebook: Boursorama-Banque-1741973942734447
+      img: Boursorama.png
```

capitalization matters.
```suggestion
      img: boursorama.png
```